File: Shinpei2-cs50-2019-x-sentimental-mario-more-20190725T150506Z/mario.py
# importing get_int
from cs50 import get_int

height = 0

# prompt to input height until height is between 1 and 8
while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break

# print("Success!", height)

# print blank and block

# print 1 record
for i in range(height):

    # print blank spaces
    blank = height - 1 - i
    for j in range(blank):
        print(" ", end="")

    # print left blocks
    for j in range(i+1):
        print("#", sep="", end="")

    # blank (2 spases)
    print("  ", end="")

    # print right blocks
    for j in range(i+1):
        print("#", sep="", end="")

    # go to a new line
    print("")

File: Shinpei2-cs50-2019-x-mario-more-20190708T082404Z/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   
    //高さを入力
    int height;
    do
    {
        height = get_int("Height: ");
    } 
    while (height < 1 || 8 < height);
    
    //ブロックを階段で表示
    for (int i = 1; i <= height ; i++)
    {
        //スペースの表示
        for (int j = 1; j <= height - i ; j++)
        {
            printf(" ");
        }
        
        //ブロックの表示
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        //空白の表示
        printf("  ");
        
        ////ブロックの表示
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        //改行
        printf("\n");
    }
}
